tools/osbuild-manifest

```diff
@@ -13,12 +13,22 @@ BOLD = "\033[1m"
 RED = "\033[31m"
 
 
-def print_pipeline(pipeline, level=0):
+def store_status(store, object_id):
+    if not store:
+        return ""
+
+    if not store.contains(object_id):
+        return ""
+
+    return " *"
```

Comment:
Awesome idea to show which objects are in the store. I'm not sure that an asterisk signals that enough, though. I only understood after reading the patch :/

Is a word `(present)` too much? Or a legend at the end of the output `*: this id exists in the store`. I'm unsure.

Reply:
Fair point! maybe `stored`? Not really having any strong opinion either way.

Reply:
`stored` works for me.